e2e/nomostest/ssh.go

```diff
@@ -112,7 +112,7 @@ func writePEMToFile(nt *NT, path, pemType string, data []byte) {
 	}
 }
 
-func createPrivateCA(nt *NT) {
+func createCaCerts(nt *NT) {
```

Comment:
this seems to be writing more than just the CACert... it's writing all sorts of things.

Also, CACert > CaCert due to Go naming conventions.

Reply:
Yeah it handles all the bootstrapping for the CA certs in the local test environment. I'll see if I can come up with a more apt name

Reply:
Done